### PR TITLE
[fix](insert) fix forget to abort txn when insert checkStrictMode failed

### PR DESCRIPTION
**Pull request by liaoxin01**
## Proposed changes

Issue Number: close #xxx

we need to abort the transaction  when checkStrictMode failed.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liaoxin01** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/AbstractInsertExecutor.java [hunk 1]
```diff
@@ -183,6 +183,8 @@ public void executeSingleInsert(StmtExecutor executor, long jobId) throws Except
         try {
             execImpl(executor, jobId);
             if (!checkStrictMode()) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/AbstractInsertExecutor.java:185]: Should use on fail not on complete
checkStrictMode should throw exception. And then go to line 205 automatically.

**Comment by liaoxin01** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**